yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,878 @@
+module vyatta-protocols-frr-isis-v1 {
+	namespace "urn:vyatta.com:mgmt:vyatta-protocols-frr-isis:1";
+	prefix vyatta-protocols-frr-isis-v1;
+
+	  import vyatta-protocols-v1 {
+		  prefix protocols;
+	  }
+  	import vyatta-services-v1 {
+    		prefix service;
+  	}
+  	import vyatta-service-snmp-v1 {
+    		prefix service-snmp;
+  	}
+  	import vyatta-types-v1 {
+    		prefix types;
+  	}
+  	import configd-v1 {
+    		prefix configd;
+  	}
+  	import vyatta-interfaces-v1 {
+    		prefix if;
+  	}
+  	import vyatta-interfaces-dataplane-v1 {
+    		prefix interfaces-dataplane;
+  	}
+  	import vyatta-interfaces-bridge-v1 {
+    		prefix interfaces-bridge;
+  	}
+
+    organization "Niral Networks Private Limited";
+    contact
+      "Niral Networks
+      Postal: Sigma Soft Tech Park, Whitefield Main Road,
+      Bengaluru, India 560066
+      Web: www.niralnetworks.com";
+
+    description
+      "Copyright (c) 2019-2020, Niral Networks Intellectual Property.
+      All rights reserved.
+
+      Redistribution and use in source and binary forms, with or
+      without modification, are permitted provided that the following
+      conditions are met:
+
+      1. Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+      2. Redistributions in binary form must reproduce the above
+      copyright notice, this list of conditions and the following
+      disclaimer in the documentation and/or other materials provided
+      with the distribution.
+      3. Neither the name of the copyright holder nor the names of its
+      contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+      THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+      'AS IS' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+      LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+      FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+      COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+      INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+      BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+      LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+      CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+      LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+      ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+      POSSIBILITY OF SUCH DAMAGE.
+
+      SPDX-License-Identifier: BSD-3-Clause
+
+      The YANG module package for vyatta-protocols-frr-isis-v1";
+
+  	revision 2018-11-15 {
+    		description "Initial revision of version 1";
+  	}
+
+
+  	typedef level {
+    		type enumeration {
+      			enum "level-1" {
+        			value 1;
```

Comment:
These values still have whitespace problems (the indentation is off by one).

Reply:
We did not update the review changes yet, will share it in sometime.